modules/etc/baseline/module-list.txt

```diff
@@ -135,7 +137,7 @@
 /org/apache/xmlbeans/main/xmlbeans-2.6.0.jar
 /org/apache/zookeeper/main/zookeeper-3.4.7.jar
 /org/beanshell/main/bsh-2.0b5.jar
-/org/bouncycastle/ext/main/bcpg-jdk15on-1.54.jar
+/org/bouncycastle/ext/main/bcpg-jdk15on-1.53.jar
```

Comment:
I assume this downgrade is a side effect of maven dependency depth and does not matter, does it?